man/man7/zpoolconcepts.7

```diff
@@ -66,13 +66,14 @@ A mirror of two or more devices.
 Data is replicated in an identical fashion across all components of a mirror.
 A mirror with
 .Em N No disks of size Em X No can hold Em X No bytes and can withstand Em N-1
-devices failing without losing data.
+devices failing, without losing data.
 .It Sy raidz , raidz1 , raidz2 , raidz3
-A variation on RAID-5 that allows for better distribution of parity and
-eliminates the RAID-5
-.Qq write hole
+A distributed-parity layout, similar to RAID-5/6, with improved distribution of
+parity, and which does not suffer from the RAID-5/6
+.Qq write hole ,
 .Pq in which data and parity become inconsistent after a power loss .
-Data and parity is striped across all disks within a raidz group.
+Data and parity is striped across all disks within a raidz group, though not
+necessarily in a consistent stripe width.
```

Comment:
The second clause added here isn't accurate.  Internally, all the dRAID raid groups will always use the same fixed stripe width regardless of the total number of physical devices.  Here's [why](https://github.com/openzfs/zfs/blob/master/module/zfs/vdev_draid.c#L45-L167) for the curious reader.  I'd suggest dropping this change.

Reply:
But that's dRAID. Isn't RAIDZ not necessarily consistent? Or do I misunderstand there?

Reply:
My mistake, with the added the phrase "A distributed-parity layout" I mistook this section for dRAID instead of RAIDZ.  You're understanding is right here.